downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/FetchTestReactomeOntologyFiles.java

```diff
@@ -0,0 +1,125 @@
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.sql.Blob;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+public class FetchTestReactomeOntologyFiles {
+	private static final Logger logger = LogManager.getLogger();
+	private static Connection connect = null;
+	private static Statement statement = null;
+	private PreparedStatement preparedStatement = null;
+	private static ResultSet resultSet = null;
+	
+	public static void execute(MySQLAdaptor dba, String username, String password, String host, String database, int releaseNumber) throws SQLException, ClassNotFoundException, UnsupportedEncodingException, FileNotFoundException, IOException {
+		logger.info("Running FetchTestReactomeOntologyFiles...");
+		Class.forName("com.mysql.jdbc.Driver");
+		connect = DriverManager.getConnection("jdbc:mysql://" + host + "/" + database + "?" + "user=" + username + "&password=" + password);
+		statement = connect.createStatement();
+		resultSet = statement.executeQuery("SELECT ontology FROM Ontology");
+		// The returned value is a single blob composed of binary and text. The three files produced by this step (pprj, pins, pont) are found within this blob.
+		// A handful of regexes and conditional statements are used to handle this data and output the 3 files. 
+		
+		PrintWriter pprjWriter = new PrintWriter("reactome_data_model.pprj");
+		PrintWriter pontWriter = new PrintWriter("reactome_data_model.pont");
+		PrintWriter pinsWriter = new PrintWriter("reactome_data_model.pins");
+		
+		while (resultSet.next()) {
+			
+			Blob blob = resultSet.getBlob("ontology");
+			InputStream is = blob.getBinaryStream();
+			BufferedReader br = new BufferedReader(new InputStreamReader(is));
+
+			String dateTime = "";
+			int dateTimeCounter = 0;
+			boolean pprjSwitch = true;
+			boolean pontSwitch = false;
+			String str;
+			logger.info("Generating reactome_data_model.pprj, reactome_data_model.pont, and reactome_data_model.pins files...");
+			while ((str = br.readLine()) != null) {
+				
+				String[] splitLine = str.split(";");
+				if (splitLine.length > 1) {
+					// A very specific regex for matching a datetime string -- Could probably be matched to a specific format that looks cleaner
+					if (splitLine[1].matches("( [A-Z][a-z]{2}){2} [0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} [A-Z]{3} [0-9]{4}")) {
+						dateTime = ";" + splitLine[1] + "\n";
+						dateTimeCounter++;
+						if (pprjSwitch) {
+							Files.write(Paths.get("reactome_data_model.pprj"), dateTime.getBytes(), StandardOpenOption.APPEND);
+						}
+						if (dateTimeCounter == 2) {
```

Comment:
What is the meaning of `dateTimeCount == 2`?

Reply:
Yes this class has a bit of a trippy structure. Each of the pprj, pont, and pins file contents are found in a single Blob output from the mysql query. The date time line was a way of dividing the file into 3 parts. 
Since the file also has some extra content we don't need, I ended up using the datetime and a regex for '[pprj|pont|pins]_file_content as a two-tiered system for getting the correct file content. I can explain this much better in person if its still confusing

Reply:
Ah, so the BLOB contains all three pieces of data, and the lines with the date-time are what separate them? Maybe just explain this with some comments.

Reply:
Yea exactly, the BLOB contains all 3 pieces of data (seperated by the datetime line), but there was also some extra in between segment 1 and 2 that we didn't need, which is why you see those 'switch' booleans. Together they're  a two-factor system that tells the line reader when to write to file

Reply:
Ok. I guess if there's no cleaner way to extract the data here, just make sure it's well documented.